eval_tasks/eval_adapter.py

```diff
@@ -27,7 +28,10 @@
 import torch
 import torch.nn.functional as F
 
+from lm_eval.models.utils import chunks
```

Comment:
Recent versions of `lm_eval` have changed the paths for many of these utility functions.